docs/getting-started/quickstart.md

```diff
@@ -0,0 +1,185 @@
+---
+slug: quickstart
+append_help_link: true
+title: Quickstart
+hide_title: false
+description: Learn how to set up Semgrep and scan your first repository.
+tags:
+  - quickstart
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Learn how to set up Semgrep, scan your first project for security issues, and view your findings.
+
+:::info prerequisites
+You must have Python 3.8 or later installed on the machine where the Semgrep CLI is running.
+:::
+
+1. Navigate to [Semgrep Cloud Platform (SCP)](https://semgrep.dev/login), and sign up by clicking on **Sign in with GitHub** or **Sign in with GitLab**. Follow the on-screen prompts to [grant Semgrep the needed permissions](/semgrep-cloud-platform/getting-started/#requested-permissions-for-github-and-gitlab) and proceed.
+
+2. Provide the **Organization display name** you'd like to use, then click **Create new organization**.
+
+3. Launch your CLI, and follow the instructions on the [**Scan a project on your machine**](https://semgrep.dev/onboarding/scan) page. For your convenience, the same information is presented below, along with instructions for Windows users.
+
+    <Tabs
+        defaultValue="macOS"
+        values={[
+        {label: 'macOS', value: 'macOS'},
+        {label: 'Linux', value: 'Linux'},
+        {label: 'Windows Subsystem for Linux (WSL)', value: 'Windows Subsystem for Linux (WSL)'},
+        {label: 'Docker', value: 'Docker'},
+        ]}
+    >
+
+    <TabItem value='macOS'>
+
+    1. Install the Semgrep CLI and confirm the installation:
```

Comment:
(Optional / no need to take any action) I can't help but notice that the spacing for the instructions seems looser than in other pages - try checking this out: [http://localhost:3000/docs/running-rules/#yaml-defined-rules][(url](http://localhost:3000/docs/running-rules/#yaml-defined-rules)) and use the inspect tool - it looks like yours is adding a p after the li, [which is totally valid html](https://stackoverflow.com/questions/7660575/is-p-semantically-correct-within-an-li-element). If it bothers you that the other lists are not <p>'ed and have tighter spacing, then you can probably remove the newline before each step, but if it looks ok to you, just leave it as is :) I'm fine with not doing any changes to the page.